source/Calamari.Tests/KubernetesFixtures/KubernetesContextScriptWrapperLiveFixture.cs

```diff
@@ -271,7 +273,14 @@ public void AuthorisingWithAmazonAccount()
             variables.Set("Octopus.Action.Kubernetes.CertificateAuthority", certificateAuthority);
             variables.Set($"{certificateAuthority}.CertificatePem", eksClusterCaCertificate);
             var wrapper = CreateWrapper();
-            TestScript(wrapper, "Test-Script");
+            
+            // When authorising via AWS, We need to make sure we are using the correct version of
+            // kubectl for the test script as newer versions may cause kubectl to fail with an error like:
+            // 'error: exec plugin: invalid apiVersion "client.authentication.k8s.io/v1alpha1"'
+            var kubectlExecutable = variables.Get(KubeCtlExecutableVariableName) ??
+                throw new Exception($"Unable to find required kubectl executable in variable '{KubeCtlExecutableVariableName}'");
```

Comment:
I am in two minds about this exception. I know why you have done this but should we just set it to "kubectl"?

Reply:
Yeah I guess we can

Reply:
I just didn't want to try and set it null if that is `variables.Get` does if the variable doesn't exist

Reply:
It feels weird though because we don't expect using `kubectl` to work in this case (well not always)... isn't it better to be explicit in the failure? I'm in two minds too haha

Reply:
Ok leave as it is with the exception